students/GuyCChan/session08/circleClass.py

```diff
@@ -0,0 +1,117 @@
+from math import pi
+
+class Circle:
+    def __init__(self, radius):
+        self.radius = radius
+
+# Chris, if I already have the properties decorator for diameter. What extra does
+# it do or what differences are there in making a classmethod for diameter?
+    @classmethod
```

Comment:
the classmethod is for making a new Circle -- the property is for getting and setting the diameter of an existing circle.